test/CMakeLists.txt

```diff
@@ -5,6 +5,8 @@ include(ExternalProject)
 find_package(SystemBLAS REQUIRED)
 
 set(SYCLBLAS_TEST ${CMAKE_CURRENT_SOURCE_DIR})
+set(CMAKE_CXX_FLAGS  "${CMAKE_CXX_FLAGS} -Wno-deprecated-declarations -Wno-deprecated-declarations -Wno-shorten-64-to-32 -Wno-cast-align") 
+set(CMAKE_CXX_FLAGS  "${CMAKE_CXX_FLAGS} -Wno-deprecated-copy-with-user-provided-copy -Wno-unused-variable")
```

Comment:
I don't mind most of these warnings but I think `-Wno-unused-variable` should be removed so that we can fix them.